installers/rpm-code/codes/flash.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+flash_main() {
+    codes_dependencies hypre
+    codes_download_proprietary "flash/FLASH-4.6.2.tar.gz" "FLASH4.6.2"
+    local flash_src_root=$PWD
+    patch_makefile
+    for n in CapLaser RTFlame
+    do
+        setup_$n $n
```

Comment:
Embed `setup_$n $n` inside make_and_install, since it's not really relevant here to know that setup exists. Quote `"setup_$n" "$n"`.

All variables in bash should be quoted except in cases where they shouldn't :)

Reply:
https://github.com/radiasoft/download/pull/95/commits/4b4294d531035f961f3421d9e7450d2b8d2c07b2